lib/moonshot/build_mechanism/github_release.rb

```diff
@@ -170,6 +169,27 @@ def fetch_changes(version)
       end
     end
 
+    # Checks for the commit's CI job status. If its not finished yet,
+    # wait till timeout.
+    #
+    # @param sha [String] Commit sha.
+    #
+    # @return [String] Status and links to the CI jobs
+    def check_ci_status(sha)
+      out = nil
+      ilog.start_threaded('Validate `CHANGELOG.md`.') do |step|
+        Retriable.retriable on: RuntimeError,
+                            max_elapsed_timeout: @ci_status_timeout,
+                            multiplier: 0,
```

Comment:
It seems like a multiplier of zero is going to make every interval zero (see https://github.com/kamui/retriable/blob/master/lib/retriable/exponential_backoff.rb#L19).